python/shipDet_conf.py

```diff
@@ -1,486 +1,906 @@
 #!/usr/bin/env python
 # -*- coding: latin-1 -*-
-import ROOT,os
+from __future__ import print_function
+from __future__ import division
```

Comment:
Why did you readd these? They should no longer have any function.

Reply:
I believe this was because I used a version before it was cleaned out. So when I rebased this popped back up.